client/components/data/query-top-posts/README.md

```diff
@@ -0,0 +1,65 @@
+Query Top Posts
```

Comment:
Indeed it would be nice, I will look into that. Is this what you are referring to in the review comment?

> Would like to see a way to easily test/try the QueryTopPosts component -- would making reviewing this easier.

Just wondering because I provided a minimal example for this purpose in the PR message. Thanks!